### DISCUSSION TITLE
Cannot extract entity sentences

### DISCUSSION THREAD

**Discussion by bharath8847** [explosion/spaCy#8928] [Help: Other Questions] [answered]
text=""" @ ! 01/02/21       10:13:46       10:21:02  
GROSS VOL Litres START 0.
GROSS VOL Litres FINISH 1000.
TICKET NUMBER                     31328.
SALE NUMBER                       30046.
DRIVER NUMBER                         1.
PRODUCT: KEROSENE        DISTILLATE     
** MULTIPLE DELIVERIES AT ONE SITE **   
 """
doc = nlp(text)
for ent in doc.ents:
    print(ent.text,ent.label_)
    print('Sentence extracted from : ', ent.sent)

**Give me this output**
Litres volume
Sentence extracted from :  None
Litres volume
Sentence extracted from :  None

I have trained my custom NER model for this for this domain specific problem

**Comment by adrianeboyd** [comment, marked as answer]
What version of spacy are you using (`spacy info`)? What components are in your pipeline (`nlp.pipe_names`)? Is there a component that sets sentence boundaries (`sentencizer`, `senter`, `parser`, custom)?

The most recent versions of spacy v2 and all versions of spacy v3 should raise an error instead of returning `None`, but I think that releases that are over a year old would return `None` if the doc doesn't have any sentence boundaries.

It's possible the rule-based `sentencizer` would work well enough for this example text because there are periods at the ends of the lines, but you may need a custom component for this kind of text, which doesn't contain the kinds of full (newspaper-ish) sentences that the `parser` has learned to segment.
